projects/plugins/jetpack/extensions/blocks/subscriptions/controls.js

```diff
@@ -240,14 +243,17 @@ export default function SubscriptionControls( {
 						}
 					} }
 				/>
-				<ToggleControl
-					disabled={ ! showSubscribersTotal }
-					label={ __( 'Include social followers in count', 'jetpack' ) }
-					checked={ includeSocialFollowers }
-					onChange={ () => {
-						setAttributes( { includeSocialFollowers: ! includeSocialFollowers } );
-					} }
-				/>
+				{ showSubscribersTotal && isPublicizeEnabled ? (
+					<ToggleControl
+						disabled={ ! showSubscribersTotal }
+						label={ __( 'Include social followers in count', 'jetpack' ) }
+						checked={ includeSocialFollowers }
+						onChange={ () => {
+							setAttributes( { includeSocialFollowers: ! includeSocialFollowers } );
+						} }
+					/>
+				) : null }
```

Comment:
This will not work for published posts on simple sites with a free plan because `isPublicizeEnabled` is more like a local state whether sharing is enabled for the post or not. For published posts, this flag is false if re-sharing is not enabled. You can check its source in [`usePublicizeConfig`](https://github.com/Automattic/jetpack/blob/e56cd77d11d243b36431ce2abaceff7d718c2eb6/projects/js-packages/publicize-components/src/hooks/use-publicize-config/index.js#L67-L69).

The correct solution here might to check if `publicize` module is active.

CC: @jeherve

Reply:
> The correct solution here might to check if publicize module is active.

This may indeed be a better solution, and we now have `useModuleStatus` to help with that. @Automattic/zap Is that something you could look into?

Thank you!